Bottom: 8aa43fb6a71227e4c5ceae602f91c240f3e6e930
Top:    05d35ec8c13c4cfdbff6d8fe1e752db1c844b9e5
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-07 00:00:00 -0500

Refresh of mirror

---

diff --git a/farm-checkout b/farm-checkout
index fa15afb..6a9a51c 100755
--- a/farm-checkout
+++ b/farm-checkout
@@ -26,6 +26,7 @@ for tag in $tags; do
 	   git clone --quiet --local --mirror $linuxgit $gittree &&
 	   git --bare --git-dir=$gittree config core.bare false &&
 	   git --bare --git-dir=$gittree config core.worktree $srctree &&
+	   cd $srctree &&
 	   git --bare --git-dir=$gittree --work-tree=$srctree checkout --quiet $tag &&
 	   echo "gitdir: $gittree" >$srctree/.git; then
 		echo -e "\e[32mOK\e[0m"
